code/modules/power/singularity/collector.dm

```diff
@@ -121,6 +131,47 @@
 	else
 		return ..()
 
+/obj/machinery/power/rad_collector/wrench_act(mob/living/user, obj/item/wrench)
+	default_unfasten_wrench(user, wrench, 0)
+	return TRUE
+
+/obj/machinery/power/rad_collector/crowbar_act(mob/living/user, obj/item/crowbar)
+	if(loaded_tank)
+		if(locked)
+			to_chat(user, "<span class='warning'>The controls are locked!</span>")
+			return TRUE
+		eject()
+		return TRUE

```

Comment:
remove this line

Reply:
Removing this line will make it so that ejecting a tank will result in the "There isn't a tank loaded!" message right after it's successfully ejected